experimental/dds/tree2/src/core/forest/forest.ts

```diff
@@ -90,6 +91,14 @@ export interface IForestSubscription extends Dependee, ISubscribable<ForestEvent
 		cursorToMove: ITreeSubscriptionCursor,
 	): TreeNavigationResult;
 
+	/**
+	 * Set `cursorToMove` to location described by path.
+	 * This is NOT a relative move: current position is discarded.
+	 * Path must point to existing node. If a destination is not provided, the cursor
+	 * is moved above the detached fields.
```

Comment:
Moving above the roots is kinda sketchy, so it warrants this extra disclaimer now that its more publicly exposed:
 ```suggestion
	 * is moved to a special dummy node above the detached fields.
	 * This dummy node can be used to read the detached fields,
	 * but other operations (such as inspecting the dummy node's type or path) should not be relied upon.
```

Reply:
Not sure why that suggestion isn't rendering properly, but I guess you can figure out what I intend

Reply:
There was a space before the ``` :)